libs/histogramming/src/HistogramsHandler.cpp

```diff
@@ -28,6 +28,7 @@ void HistogramsHandler::SetupHistograms() {
   for (auto &[name, title] : histTitles) {
     histograms1D[name] = new TH1D(title.c_str(), title.c_str(), histNbins[name], histMin[name], histMax[name]);
   }
+  histograms1D["cutFlow"] = new TH1D();
```

Comment:
The previous comment is especially true if we hardcode "cutFlow" histogram here. Of course, we could simply not use it, but it suggests it's some non-user customizable histogram that should always be filled.